libs/api/books/src/lib/reading-list.service.ts

```diff
@@ -28,4 +28,17 @@ export class ReadingListService {
       return list.filter(x => x.bookId !== id);
     });
   }
+
+  //TODO: book reading finished
+  async bookRedingFinished(id: string): Promise<void> {
+    this.storage.update(list => {
```

Comment:
revisit this logic, this is more complex logic

use `map` function to handle this